### ISSUE TITLE
Add a horizon chart

### ISSUE THREAD

**Issue by ColinEberhardt** [ScottLogic/d3fc#580]
See: https://public.tableau.com/s/gallery/unemployment-horizon-chart

**Comment by bjedrzejewski** [comment]
Is that solvable as a special case of small multiples chart?

**Comment by ColinEberhardt** [comment]
certainly, yes. I see this as small-multiples component being used in conjunction with a 'horizon series' component.
